shared_model/interfaces/base/signable.hpp

```diff
@@ -88,6 +87,29 @@ namespace shared_model {
             and this->createdTime() == rhs.createdTime();
       }
 
+/**
```

Comment:
Note: copied from https://github.com/hyperledger/iroha/pull/972/files#diff-f5da1abc9499e5ab936bf6172f85dbb6R95.

Reply:
Maybe lets wait until that code that code is accepted, before merging this PR?

Reply:
@Alexey-N-Chernyshov  can resolve this conflict when he will be merging his PR if he does not mind. What do you think, @Alexey-N-Chernyshov

Reply:
ok, i don't mind